src/components/modal/Modal.scss

```diff
@@ -133,3 +135,33 @@
     z-index: 89;
     display: flex;
 }
+
+//Responsive support
+.is-responsive-web {
+    .modal {
+        @include breakpoint($modal-fullscreen) {
+            padding: 0;
+        }
+    }
+
+    .modal-dialog-container {
+        width: 100%;
+        margin: auto;
+
+        @include breakpoint($modal-fullscreen) {
+            height: 100%;
+
+            .modal-dialog {
```

Comment:
Does this need to be nested inside of the already-nested `.modal-dialog-container` rule? We should aim for as little specificity as is needed to achieve the desired effect.

Reply:
Refactored to reduce specificity.